### PR TITLE
DOCSP-43077: Specify Documents to Return

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-43077>
Staging - <https://preview-mongodbmcmorisi.gatsbyjs.io/c/DOCSP-43077-specify-documents/read/specify-documents-to-return/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/specify-documents-to-return.txt [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+.. _c-limit:
+
+Limit
+-----
+
+To specify the maximum number of documents returned from a read operation,
+specify the ``limit`` option to your ``mongoc_collection_find_with_opts()``
```
Comment (jordan-smith721) [inline source/read/specify-documents-to-return.txt:44]: I think "pass" works better here. It also gets rid of the repeated "specify"
```suggestion
pass the ``limit`` option to your ``mongoc_collection_find_with_opts()``
```

File: source/read/specify-documents-to-return.txt [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+   section describes how to return the documents in a specified sort order.
+
+.. _c-sort:
+
+Sort
+----
+
+To return documents in a specified order, use the ``sort`` option. The ``sort``
+option takes a sort direction as a parameter. To specify the sort direction,
+pass a ``1`` to specify an ascending sort or a ``-1`` to specify a descending sort.
```
Comment (jordan-smith721) [inline source/read/specify-documents-to-return.txt:81]:
```suggestion
pass a ``1`` for an ascending sort or a ``-1`` for a descending sort.
```

File: source/read/specify-documents-to-return.txt [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+.. _c-skip:
+
+Skip
+----
+
+To skip a specified number of documents before returning your query results,
+use the ``skip`` option and pass in the number of documents to skip. The
+``skip`` option ignores the specified number of documents in your query
+results and returns the rest.
```
Comment (jordan-smith721) [inline source/read/specify-documents-to-return.txt:115]:
```suggestion
results, then returns the rest.
```

**Review by jordan-smith721 — APPROVED** [review]
Nice! Just a couple small things but LGTM

File: source/includes/read/specify-documents-to-return.c [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    }
+    {
+        // Fetches all documents with a cuisine field value of "Italian" and returns them in ascended sort order
+        // start-sort
+        const bson_t *doc;
+
+        bson_t *filter = BCON_NEW ("cuisine", BCON_UTF8 ("Italian"));
+        bson_t *opts = bson_new ();
+        bson_t *sort = BCON_NEW ("name", BCON_INT32 (1));
+        BSON_APPEND_DOCUMENT (opts, "sort", sort);
```
Comment (kevinAlbs) [inline source/includes/read/specify-documents-to-return.c:47]: To shorten:

```suggestion
        bson_t *opts = BCON_NEW("sort", "{", "name", BCON_INT32(1), "}");
```

File: source/includes/read/specify-documents-to-return.c [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    }
+    {
+        // Fetches all documents with a cuisine field value of "Italian", skips the first 10 results, limits the number of returned results to 5, and sorts them in ascending order
+        // start-limit-sort-skip
+        const bson_t *doc;
+
+        bson_t *filter = BCON_NEW ("cuisine", BCON_UTF8 ("Italian"));
+        bson_t *opts = BCON_NEW ("limit", BCON_INT64 (5), "skip", BCON_INT64 (10));
+        bson_t *sort = BCON_NEW ("name", BCON_INT32 (1));
+        BSON_APPEND_DOCUMENT (opts, "sort", sort);
```
Comment (kevinAlbs) [inline source/includes/read/specify-documents-to-return.c:94]:
```suggestion
        bson_t *opts = BCON_NEW("limit", BCON_INT64(5), "skip", BCON_INT64(10), "sort", "{", "name", BCON_INT32(1), "}");
```

**Review by kevinAlbs — APPROVED** [review]
LGTM with a possible shortening of example code.

**Pull request merged**